app/models/form/lettings/subsections/setup.rb

```diff
@@ -10,14 +10,22 @@ def pages
     @pages ||= [
       organisation_page,
       stock_owner_page,
+      Form::Lettings::Pages::MinRentValueCheck.new(nil, nil, self, check_answers_card_number: nil),
```

Comment:
You could just make the default value nil here, if required.

Reply:
Yeah, i think there's a tendency to do this more explicitly but I'm not that sold on it myself, will make this change